src/js/background/background.js

```diff
@@ -9,8 +9,8 @@ browser.runtime.onInstalled.addListener(async () => {
   if (firstRunShown) {
     return;
   }
-  const userApiToken = await browser.storage.local.get("apiToken");
```

Comment:
Question: Do we need to do any sort of cleanup/removal/migration for the previous `userApiToken` storage item? Will users be logged out of the Relay add-on when this lands? Do we need to ship something to the website first?